data/moves.js

```diff
@@ -7146,14 +7143,7 @@ let BattleMovedex = {
 		priority: 0,
 		flags: {},
 		isMax: "Orbeetle",
-		secondary: {
-			chance: 100,
-			self: {
-				onHit() {
-					this.field.addPseudoWeather('gravity');
-				},
-			},
-		},
+		pseudoweather: 'gravity',
```

Comment:
```suggestion
		pseudoWeather: 'gravity',
```

Reply:
Huh, and I thought I'd tested all of the moves. I must be blind...